src/understanding-workflow/concepts.md

```diff
@@ -108,11 +108,11 @@ Note that during the deploy, your application will be running in a mixed mode.
 
 Workflow treats all persistent services such as databases, caches, storage,
 messaging systems, and other [backing services][] as resources managed
-separtely from your application. This philosophy aligns with Twelve-Factor
+separately from your application. This philosophy aligns with Twelve-Factor
```

Comment:
There is such a link above... and another one below.  IMO, with three tightly grouped mentions of any one things (such as is the case here), I'd have only the first occurrence be a link... otherwise, it somehow manages to look obnoxious.